erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -580,11 +577,16 @@ def update_stock_entry_based_on_material_request(pick_list, stock_entry):
 
 	return stock_entry
 
-def update_stock_entry_items_with_no_reference(pick_list, stock_entry):
-	for location in pick_list.locations:
-		item = frappe._dict()
-		update_common_item_properties(item, location)
-
+def update_stock_entry_items_with_no_reference(location, stock_entry):
+	item = frappe._dict()
+	update_common_item_properties(item, location)
+	if location.source_package_tag:
+		item.package_tag = location.source_package_tag
+		# item.s_warehouse = location.warehouse
```

Comment:
@neilLasrado Currently we cannot have target warehouse and source warehouse as same. so which warehouse going to get selected as a target while `Repack` is made?  also `Pick List` is created for `Delivery`. so technically the packages are not relocated from one warehouse to other. it is going out to the Customer. so making entry in Target warehouse has no meaning. please explain how can we deal with this.

Reply:
@bhattdevarsh is this still an issue?